flake.nix

```diff
@@ -263,9 +265,9 @@
               };
 
           benchmarks = craneLib.mkCargoDerivation {
-            inherit src version cargoArtifacts env;
+            inherit pname src version cargoArtifacts env;
 
-            pname = "nickel-lang-bench";
+            pnameSuffix = "-bench";
```

Comment:
Does this make any difference with respect to building? For example, why using `pnameSuffix` here but setting a complete name in other places (`nickel-lang-cli`, `nickel-lang-cli-static`, etc.)?